FreeTAKServer/controllers/certificate_generation.py

```diff
@@ -1,25 +1,27 @@
 # !/usr/bin/python
-import subprocess
-try:
-    from OpenSSL import crypto
-except ImportError:
-    subprocess.run(["pip3", "install", "pyopenssl"], capture_output=True)
-from OpenSSL import crypto
 import os
-import getopt
 import sys
+import getopt
 import random
-from shutil import copyfile
 import uuid
-from jinja2 import Template
 import socket
 import zipfile
 import shutil
 import pathlib
+
+from shutil import copyfile
+from jinja2 import Template
 from FreeTAKServer.controllers.configuration.MainConfig import MainConfig
 
+try:
+    from OpenSSL import crypto
+except ImportError:
+    print("Failed to load OpenSSL. (hint: install pyopenssl, `python -m pip install pyopenssl`)")
+    sys.exit(2)
+
+
```

Comment:
Did a slight refactor here since self modifying the environment is super awkward.

Reply:
that's fair.